libwallet/instantswap.go

```diff
@@ -311,15 +313,3 @@ func (mgr *AssetsManager) IsOrderSchedulerRunning() bool {
 func (mgr *AssetsManager) GetShedulerRuntime() string {
 	return time.Since(mgr.InstantSwap.SchedulerStartTime).Round(time.Second).String()
 }
-
-// MarketName is a convenience function that returns a proper market name for
-// the from and to currencies used when fetching ticker data from binance.
-func MarketName(fromCur, toCur string) string {
-	dcrToLtc := strings.EqualFold(fromCur, "dcr") && strings.EqualFold(toCur, "ltc")
-	btcToOthersExceptUsdt := strings.EqualFold(fromCur, "btc") && !strings.EqualFold(toCur, "usdt")
-	if dcrToLtc || btcToOthersExceptUsdt {
-		return toCur + "-" + fromCur // e.g DCR/BTC, LTC/BTC and not BTC/LTC or BTC/DCR
-	}
-
-	return fromCur + "-" + toCur
-}
```

Comment:
@ukane-philemon i see we had a function that did this already, why was this deprecated?

Reply:
Yes. We now have an improved version of this function in the libwallet/ext/rate_source.go file (isSupportedMarket).